### ISSUE TITLE
[FEATURE REQUEST] Glyph of Dark Succor on DK

### ISSUE THREAD

**Issue by Pocokk** [ennvina/spellactivationoverlay#316]
Hey there!

I would like to you add the following effect as a glowing action bar icon on Death Strike, whenever you get this buff:

https://www.wowhead.com/cata/spell=96279/glyph-of-dark-succor

https://www.wowhead.com/cata/spell=101568/dark-succor

Thank you!

**Comment by ennvina** [comment]
Hi @Pocokk and thank you for your feedback.

It looks like a good candidate, but there is a problem: it require tracking 2 buffs at the same time i.e., the proc from the glyph and the Presence you are in. Unfortunately, the addon currently doesn't support multi-buff procs.

There may be a solution by using "shapeshift forms" but it requires additional development.

TL;DR: it is possible to add it, but not obvious at the moment. Stay tuned!

**Comment by Pocokk** [comment]
Thank you, patiently waiting the end result!

**Comment by ennvina** [comment]
Implemented in #376 and available officially since v2.4.2. Thank you for your patience.

**Comment by Pocokk** [comment]
Thank you dear ennvina!
